src/main/java/space/npstr/magma/connections/hax/ClosingUndertowWebSocketClient.java

```diff
@@ -56,8 +58,11 @@ public ClosingUndertowWebSocketClient(final XnioWorker worker,
         final MonoProcessor<Void> completion = MonoProcessor.create();
         return Mono.fromCallable(
                 () -> {
-                    WebSocketClient.ConnectionBuilder builder = this.createConnectionBuilder(url);
-                    List<String> protocols = this.beforeHandshake(url, headers, handler);
+                    if (logger.isDebugEnabled()) {
+                        logger.debug("Connecting to " + url);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Format specifiers should be used instead of string concatenation. ([squid:S3457](https://sonarcloud.io/organizations/napstr-github/rules#rule_key=squid%3AS3457))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=space.npstr.magma%3Amagma&pullRequest=14&issues=AWas-MiDFsWw174ZRTe9&open=AWas-MiDFsWw174ZRTe9)

[ISSUE_KEY]: AWas-MiDFsWw174ZRTe9